src/scraper/run.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require_once '../../vendor/autoload.php';
+
+$dbConnection = new TopDog\Classes\DbConnection();
+$ch = new TopDog\scraper\Classes\CurlHandler();
+$request = new TopDog\scraper\Classes\APIGrabber($ch);
+$dataProcessor = new TopDog\scraper\Classes\DataProcessor($request);
+$dbHandler = new TopDog\Classes\DbHandler($dbConnection);
+$arrayBreeds = $dataProcessor->processBreedData();
+foreach ($arrayBreeds as $breed) {
```

Comment:
potentially move all this stuff into top level class

Reply:
(everything that can be from this line to the bottom)

Reply:
James